lib/eofs/standard.py

```diff
@@ -152,7 +158,18 @@ def __init__(self, dataset, weights=None, center=True, ddof=1):
         dataNoMissing = self._data[:, nonMissingIndex]
         # Compute the singular value decomposition of the design matrix.
         try:
-            A, Lh, E = np.linalg.svd(dataNoMissing, full_matrices=False)
+            if has_dask and isinstance(dataNoMissing, dask.array.Array):
+                # Use the parallel Dask algorithm
+                dsvd = dask.array.linalg.svd(dataNoMissing)
+                A, Lh, E = (x.compute() for x in dsvd)
+            else:
+                # Basic numpy algorithm
+                A, Lh, E = np.linalg.svd(dataNoMissing)
```

Comment:
This can be way more expensive for numpy arrays than with `full_matrices=False`, I don't think it is a good idea to change this.

Reply:
No problem, changed back to the original and moved the array trim to the dask section